nautobot_golden_config/nornir_plays/config_compliance.py

```diff
@@ -6,6 +6,7 @@
 from collections import defaultdict
 from datetime import datetime
 from django.utils.timezone import make_aware
+from lxml import etree  # nosec
```

Comment:
this is considered insecure? what are the other options?

Reply:
just following up on this one.

Reply:
We just need to understand what crypt package lxml is using and how. If it is strictly for non-security related hashing then #nosec is fine.

Reply:
Is this still outstanding?

Reply:
@jeffkala @itdependsnetworks @nkallergis Could you please weigh in on this,  do we consider the backup and intended repo's trusted xml?

Reply:
I am good with: https://github.com/nautobot/nautobot-app-golden-config/pull/708/commits/0091ceb1a314cd9e37bc8482bcdda6b920a4bde1#diff-c907c96f839747adca2132acff8482da45ac09b6f2cafa0faa0e57f8395874e5R11